web/admin/utils/loadCompaniesData.js

```diff
@@ -12,9 +13,12 @@ export async function loadCompaniesData(api, userId, minDate, companyId) {
     }
   );
 
-  const companyIds = companyId
-    ? [companyId]
-    : [companiesListResponse.data.user.adminedCompanies[0].id];
+  const sortedCompaniesByName = sortBy(
+    companiesListResponse.data.user.adminedCompanies,
+    "name"
+  );
```

Comment:
let's say i have VIR 8 / VIR 9 / VIR 10, which one would be the first if I sort by name ? VIR 10 or VIR 8 ?

Reply:
I guess VIR 10 would be before VIR 8. Not ideal but acceptable I guess.